{{cookiecutter.project_slug}}/tox.ini

```diff
@@ -6,25 +6,15 @@ skipsdist = True
 
 [testenv]
 setenv =
-    DJANGO_SETTINGS_MODULE={{ cookiecutter.project_slug }}.settings.base
+    DJANGO_SETTINGS_MODULE = {{ cookiecutter.project_slug }}.config.settings.test
     PYTHONPATH={toxinidir}
-    SECRET_KEY=test
     DATABASE_URL=postgresql://{{ cookiecutter.project_slug }}:{{ cookiecutter.project_slug }}@localhost/{{ cookiecutter.project_slug }}
-deps = -r{toxinidir}/requirements/base.txt
+deps = -r{toxinidir}/requirements/dev.txt
```

Comment:
Any specific reason? Thing is, I often run pytest manually while developing, especially because I need to change the args (-s, -k, -x etc.). I would then have to install test.txt in my dev env, which kinda makes dev.txt not useful...